### ISSUE TITLE
Vesuvio - Intensity constraints not functioning as expected

### ISSUE THREAD

**Issue by ElliotAOram** [mantidproject/mantid#16039]
### Expected behavior
* The intensity of the peaks in the Vesuvio fitting should be as described in the `[intensity_constraints']` flag in the driver script

### Actual behavior
* The second fit in VesuvioTOFFit that is used to obtain the Chi squared value appears to change the values that should be constrained 

### Steps to reproduce the behavior
* Run a complex intensity constrained vesuvio fit e.g.
`flags['intensity_constraints'] = list([2.7593,-0.6864,0,0,0]) , list([0,0,0,2.1580,-0.4968]) ,  list([0,0.4968,0,-2.7593,0])`

### Platforms affected
* All

**Comment by martyngigg** [comment]
@ElliotAOram I've had to revert the merge to master as all of the clean builds broke. We don't yet understand why the pull request builds didn't pick this up.

If you download [vesuvio_intensity_patch.txt](https://github.com/mantidproject/mantid/files/262672/vesuvio_intensity_patch.txt) and switch to the top of the mantid source on your machine you can get the changes back onto a new branch by running:

````
git co --no-track -b 16039_vesuvio_intensity_constraints_fixed
git apply vesuvio_intensity_patch.txt
````

**Comment by ElliotAOram** [comment]
This issue will now close #15600

**Comment by ElliotAOram** [comment]
All Vesuvio tests passed locally on windows 7 and Ubuntu 14.04

**Comment by ElliotAOram** [comment]
Merged with release branch for 3.7 hence can be closed
